checker-qual/src/main/java/org/checkerframework/framework/qual/DefaultQualifier.java

```diff
@@ -63,6 +66,13 @@
      */
     TypeUseLocation[] locations() default {TypeUseLocation.ALL};
 
+    /**
+     * When used on a package, whether the defaults should also apply to subpackages.
+     *
+     * @return whether the default should be inherited by subpackages
+     */
+    boolean applyToSubpackages() default true;
```

Comment:
I think the following is fine, but to write it down explicitly as a reminder to myself:

This means that we'll need to change not just to the eisop checker but also to the eisop version of checker-qual.

Again, that should be fine.